templates/data_management/index.html

```diff
@@ -33,12 +33,12 @@ <h2>External Objects</h2>
     data-ajax="externalObjectsData"
     data-pagination="true"
     data-side-pagination="server"
-    data-sort-name="doi_or_unique_name"
+    data-sort-name="other_unique_name"
```

Comment:
If this is a default sort order, should it be `identifier`?

Reply:
Resolved in https://github.com/FAIRDataPipeline/data-registry/commit/332d72571806345b31ebe68e1aa2a08085ce2e70